stores/moderationSubmissionsStore.ts

```diff
@@ -17,6 +17,22 @@ export const useModerationSubmissionsStore = defineStore(
         const selectedSubmissionId: Ref<string> = ref('')
         const selectedSubmissionData: Ref<Submission | undefined> = ref()
         const filteredSubmissionDataForListComponent: Ref<Submission[]> = ref([])
+        const updateSubmissionFromTopNav: Ref<boolean> = ref(false)
+        const isSendingUpdatedForm: Ref<boolean> = ref(false)
+        const didMutationFail: Ref<boolean> = ref(false)
+        const mutationAttempts: Ref<number> = ref(0)
+
+        function handleUpdateSubmissionFromTopNav(newValue: boolean) {
+            updateSubmissionFromTopNav.value = newValue
+        }
+
+        function handleUpdateIsSendingUpdatedForm(newValue: boolean) {
```

Comment:
these handles aren't really clear what they're handling. Usually `handle` is used for handling an event. In this case, these just look like `setX` functions